test/e2e/storage/sanity.go

```diff
@@ -561,9 +561,13 @@ func (v volume) create(sizeInBytes int64, nodeID string) (string, *csi.Volume) {
 			},
 		}
 	}
-	vol, err := v.cc.CreateVolume(
-		v.ctx, req,
-	)
+	var vol *csi.CreateVolumeResponse
+	err = v.retry(func() error {
```

Comment:
@pohly I guess this change might affect other sanity tests, not just stress test. Is that expectation?

Reply:
That's a good point. It was intended that they also retry, however, there's one small problem with that: the default context in other tests doesn't have a timeout. That needs to be added, otherwise permanent errors cause retries until the entire test suite is killed.

Reply:
I've added a default timeout of 5 minutes.